README.md

```diff
@@ -21,9 +21,16 @@ Make sure you are using the lastest version of Raspbian:
     $ sudo apt-get update
     $ sudo apt-get upgrade
 
-Install `pifacecommon` (for Python 3 and 2) with the following command:
+Install `pifacecommon` with the following commands:
 
-    $ sudo apt-get install python{,3}-pifacecommon
+    Python 2:
+    $ sudo pip pifacecommon
```

Comment:
I'd remove this, Python 2 was sunset Jan 1 2020 and should no longer be used. https://www.python.org/doc/sunset-python-2/